public/server/db/gn.json

```
@@ -0,0 +1,4 @@
+{
+  "definition": "Good Night",
+  "alternatives": "Get Naked
```

You missed a quote at the end here.
```suggestion
  "alternatives": "Get Naked"
```